crates/tuono/src/scaffold_project.rs

```diff
@@ -265,14 +285,12 @@ mod tests {
     use super::*;
     #[test]
     fn generate_valid_content_url_from_head() {
-        let expected = format!(
-            "{}/{}/{}",
-            GITHUB_RAW_CONTENT_URL, "main", "examples/tuono-app"
-        );
+        let expected = format!("{}/{}/{}", "localhost:3000", "main", "examples/tuono-app");
```

Comment:
```suggestion
        let expected = format!("{}/{}/{}", "http://localhost:3000", "main", "examples/tuono-app");
```

Do these URLs need protocols?